CHANGELOG-1.4.md

```diff
@@ -177,15 +177,15 @@ filename | sha256 hash
 ### Other notable changes
 
 * kube-apiserver now drops unneeded path information if an older version of Windows kubectl sends it. ([#44586](https://github.com/kubernetes/kubernetes/pull/44586), [@mml](https://github.com/mml))
-* Bump gcr.io/google_containers/glbc from 0.8.0 to 0.9.2. Release notes: [0.9.0](https://github.com/kubernetes/ingress/releases/tag/0.9.0), [0.9.1](https://github.com/kubernetes/ingress/releases/tag/0.9.1), [0.9.2](https://github.com/kubernetes/ingress/releases/tag/0.9.2) ([#43098](https://github.com/kubernetes/kubernetes/pull/43098), [@timstclair](https://github.com/timstclair))
+* Bump k8s.gcr.io/glbc from 0.8.0 to 0.9.2. Release notes: [0.9.0](https://github.com/kubernetes/ingress/releases/tag/0.9.0), [0.9.1](https://github.com/kubernetes/ingress/releases/tag/0.9.1), [0.9.2](https://github.com/kubernetes/ingress/releases/tag/0.9.2) ([#43098](https://github.com/kubernetes/kubernetes/pull/43098), [@timstclair](https://github.com/timstclair))
```

Comment:
Is it worth updating historial changelogs?

Reply:
I waffle on this.  The image may eventually cease to exist at the old URL?  Or maybe we simply say it won't?

Reply:
My thought is, the application configuration and manifests for, say 1.4, will still be looking for the image at `gcr.io/google_containers/glbc`. Modifying the changelog, doesn't go back and create a new patch release of 1.4 updating the config/manifests to look at the new place.

I'd say it would be worth doing for any version we are actually going to go back and change. If this is just for 1.10 forward, then I'd suggest leaving the historical change logs be.

Reply:
ACK